diff -Nur old/AMAVIS.pm new/AMAVIS.pm
--- old/AMAVIS.pm	2004-07-14 14:03:53.000000000 +0000
+++ new/AMAVIS.pm	2004-07-14 14:11:40.000000000 +0000
@@ -96,7 +96,7 @@
 	   );
 
 
-$conffile = '/etc/amavis-ng/amavis.conf';
+$conffile = '/etc/amavis.conf';
 
 sub init {
   my $self = shift;
diff -Nur old/doc/amavis-ng.texi new/doc/amavis-ng.texi
--- old/doc/amavis-ng.texi	2003-04-14 15:18:47.000000000 +0000
+++ new/doc/amavis-ng.texi	2004-07-22 21:35:13.000000000 +0000
@@ -335,7 +335,7 @@
 @end menu
 
 All configuration parameters are stored in
-@file{/etc/amavis-ng/amavis.conf} which has a INI-style layout. The
+@file{/etc/amavis.conf} which has a INI-style layout. The
 @command{--configfile} parameter can be used to specify a different
 file.
 
diff -Nur old/etc/amavis.conf new/etc/amavis.conf
--- old/etc/amavis.conf	2003-04-08 14:09:56.000000000 +0000
+++ new/etc/amavis.conf	2004-07-15 14:39:25.000000000 +0000
@@ -78,7 +78,7 @@
 
 ;; Where should mails be unpacked to?
 
-; unpack dir = /tmp/
+unpack dir = /var/spool/amavis-ng/tmp
 
 ;; Where should infected mail be stored? (Only the actual mail, not
 ;; the unpacked attachments)
@@ -224,21 +224,15 @@
 
 ;; Free (un)compression programs
 bzip2 = /usr/bin/bzip2
-arj = /usr/bin/arj
-nomarch = /usr/bin/nomarch
 gzip = /bin/gzip
-unzoo = /usr/bin/unzoo
+arc = /usr/bin/arc
+zoo = /usr/bin/zoo
+unarj = /usr/bin/unarj
 
 ;; lha, unrar are non-free.
 ; lha = /usr/bin/lha
 ; unrar = /usr/bin/unrar
 
-;; For arc zoo, unarj, there are free alternatives whcih are enabled
-;; above.
-; arc = /usr/bin/arc
-; zoo = /usr/bin/zoo
-; unarj = /usr/bin/unarj
-
 [security]
 
 ;; Resource limits for unpacking each message
