diff -Naur amavis-ng-0.1.4.1.orig.orig/Makefile.PL amavis-ng-0.1.4.1.orig/Makefile.PL
--- amavis-ng-0.1.4.1.orig.orig/Makefile.PL	Thu Aug 29 16:57:56 2002
+++ amavis-ng-0.1.4.1.orig/Makefile.PL	Thu Aug 29 17:55:07 2002
@@ -21,18 +21,18 @@
 	  \$(INSTALLSCRIPT)\/..\/share\/amavis\/magic.mime
 	# Install AMaViS main script
 	install -D -m755 amavis \$(INSTALLSCRIPT)\/amavis
-	install -D -m4755 -omail -gmail qmail-queue.amavis \\
+	install -D -m4755 qmail-queue.amavis \\
 		\$(INSTALLSCRIPT)\/..\/sbin\/qmail-queue.amavis
-	install -D -m755 amavis-milter\/amavis-milter \\
-		\$(INSTALLSCRIPT)\/..\/sbin\/amavis-milter
+#	install -D -m755 amavis-milter\/amavis-milter \\
+#		\$(INSTALLSCRIPT)\/..\/sbin\/amavis-milter
 	# Install spool directories
-	install -d -m750 -omail -gmail \\
-		\$(PREFIX)\/..\/var\/spool\/amavis\/quarantine \\
-		\$(PREFIX)\/..\/var\/spool\/amavis\/problems \\
-		\$(PREFIX)\/..\/var\/spool\/amavis\/queue
-	install -d -m750 -omail -gmail \\
-		\$(PREFIX)\/..\/var\/log\/amavis \\
-		\$(PREFIX)\/..\/var\/run\/amavis
+	install -d -m750 \\
+		\$(DESTDIR)\/\$(PREFIX)\/..\/var\/spool\/amavis\/quarantine \\
+		\$(DESTDIR)\/\$(PREFIX)\/..\/var\/spool\/amavis\/problems \\
+		\$(DESTDIR)\/\$(PREFIX)\/..\/var\/spool\/amavis\/queue
+	install -d -m750 \\
+		\$(DESTDIR)\/\$(PREFIX)\/..\/var\/log\/amavis \\
+		\$(DESTDIR)\/\$(PREFIX)\/..\/var\/run\/amavis
 	  /;
   $inherited;
 }
diff -Naur amavis-ng-0.1.4.1.orig.orig/etc/amavis.conf amavis-ng-0.1.4.1.orig/etc/amavis.conf
--- amavis-ng-0.1.4.1.orig.orig/etc/amavis.conf	Thu Aug 29 16:57:56 2002
+++ amavis-ng-0.1.4.1.orig/etc/amavis.conf	Thu Aug 29 17:51:04 2002
@@ -182,8 +182,8 @@
 maxspace = 30M
 
 ;; If amavis is run as UID root, drop root privileges to uid, gid.
-uid = mail
-gid = mail
+uid = amavis 
+gid = amavis
 
 [FSAV]
 
@@ -235,4 +235,4 @@
 
 [OpenAntiVirus]
 
-socket = localhost:8127
\ No newline at end of file
+socket = localhost:8127
